src/internet_identity/src/http.rs

```diff
@@ -7,24 +7,24 @@ use serde_bytes::ByteBuf;
 
 mod metrics;
 
-pub fn http_request(req: HttpRequest) -> HttpResponse {
-    let parts: Vec<&str> = req.url.split('?').collect();
+fn http_options_request() -> HttpResponse {
+    // TODO: Restrict origin to just the II-specific origins.
+    let headers = vec![("Access-Control-Allow-Origin".to_string(), "*".to_string())];
+
+    HttpResponse {
+        // Indicates success without any additional content to be sent in the response content.
+        status_code: 204,
+        headers,
+        body: ByteBuf::from(vec![]),
+        upgrade: None,
+        streaming_strategy: None,
+    }
+}
+
+fn http_get_request(url: String, certificate_version: Option<u16>) -> HttpResponse {
+    let parts: Vec<&str> = url.split('?').collect();
+
     match parts[0] {
-        // The FAQ used to live in '/faq' but we now use an external website. We redirect in order to not
-        // break existing links in the wild.
-        "/faq" => HttpResponse {
```

Comment:
So, instead of managing the `/faq` in the backend, you moved to redirecting from the frontend, is that right?